Hakawai/Mentions/HKWMentionsPlugin.m

```diff
@@ -844,7 +844,10 @@ - (BOOL)advanceStateForCharacterInsertion:(unichar)newChar
         case HKWMentionsStateQuiescent: {
             // Inform the start detection state machine that a character was inserted. Also, override the double space
             //  to period auto-substitution if the substitution would place a period right after a preceding mention.
-            [self.startDetectionStateMachine characterTyped:newChar asInsertedCharacter:NO previousCharacter:precedingChar];
+
+            // When control character is inserted before existing word in text view, then query mention with that word.
+            NSString *textAfterControlCharacter = [HKWMentionsStartDetectionStateMachine wordAfterLocation:location text:parentTextView.text];
```

Comment:
doesn't using this break down some core elements of the hakawai state machine? like if we just look up the word in each moment, rather than using the set location and prev/next chars, etc. that are being set throughout the code? Let's talk on slack

Reply:
Here we want to achieve functionality where mentioned should be detected even after inserting @ after some name.
We will ping you over slack

Reply:
you don't know where this is a control character yet, so update the comment and call this wordFollowingTypedCharacter

Reply:
Updated comment and renamed `textAfterControlCharacter`.